tests/logs.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from logging.handlers import RotatingFileHandler
+from pathlib import Path
+
+import structlog
+
+
+def find_project_dir(path: Path) -> Path:
+    if path.is_dir():
+        for contained_path in path.iterdir():
+            if contained_path.name == "pyproject.toml":
+                return path
+    return find_project_dir(path.parent)
+
+
+def setup_log(log_file_name: str) -> None:
```

Comment:
This mostly follows [the advice in structlog's docs](https://www.structlog.org/en/stable/standard-library.html) and it looks pretty good, but it would be really nice if all logs, including logs written to the standard logger, were formatted as JSON consistently.

Can you go farther, [look at this section of the docs](https://www.structlog.org/en/stable/standard-library.html#rendering-using-structlog-based-formatters-within-logging), and make sure that everything is logged in JSON format? This could be useful if we need to debug behavior in third-party libraries we don't control.